backend/prisma/schema.prisma

```diff
@@ -8,20 +8,38 @@ datasource db {
 }
 
 model User {
-  id           String    @id @default(uuid())
-  email        String   @unique
-  firstName    String
-  lastName     String
-  password     String?
-  birthDate    DateTime?
-  googleID     String?   @unique
+  UserID      String             @id @default(uuid())
```

Comment:
generally it's normal to leave the id field of the table to simply be `id`, and it's somewhat a convention that when it's a FK to name it based on the table